README.md

```diff
@@ -151,20 +175,53 @@ It means the FluentBit Operator works properly if you see the above messages, yo
 kubectl delete -f https://raw.githubusercontent.com/kubesphere/fluentbit-operator/master/manifests/quick-start/quick-start.yaml
 ```
 
-### Configure Custom Watch Namespaces
+##### Fluentd quick start
 
-When starting the operator, you can pass an optional set of namespaces to watch for resources in with the `--watch-namespaces` flag. It takes a comma-separated list of namespaces to watch:
+Fluentd can receives logs from Fluent Bit, both needs to enable the forward plugin.
```

Comment:
Fluentd in Fluent Operator is used to receive logs from Fluent Bit or other sources like HTTP and Syslog, etc.